.devcontainer/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# OS
+FROM mcr.microsoft.com/devcontainers/python:3.11
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && apt-get install -y firefox-esr libgtk-3-dev
```

Comment:
What are `firefox-esr` and `libgtk-3-dev` used for within the dev environment? This step also took a long time to build the container, so maybe we could get rid of this entire line? If we should keep it you should add a comment describing what they are used for otherwise someone might come and try to slim it down later because they forgot what it was needed for (i.e. me ;) )

Reply:
libgtk-3-dev is used for graphics, in case we want to connect to vscode and plot stuff.  

firefox-esr is the firefox web browser.  I was experimenting with one of my plotting libraries, pytplot, which can make HTML plots using bokeh.  We don't necessarily need it though.  I think it's worthwhile to keep libgtk-3-dev, but maybe get rid of firefox-esr since I doubt others will be using it.